// stoker/dist/http-status-codes.cjs
'use strict';

const ACCEPTED = 202;
const BAD_GATEWAY = 502;
const BAD_REQUEST = 400;
const CONFLICT = 409;
const CONTINUE = 100;
const CREATED = 201;
const EXPECTATION_FAILED = 417;
const FAILED_DEPENDENCY = 424;
const FORBIDDEN = 403;
const GATEWAY_TIMEOUT = 504;
const GONE = 410;
const HTTP_VERSION_NOT_SUPPORTED = 505;
const IM_A_TEAPOT = 418;
const INSUFFICIENT_SPACE_ON_RESOURCE = 419;
const INSUFFICIENT_STORAGE = 507;
const INTERNAL_SERVER_ERROR = 500;
const LENGTH_REQUIRED = 411;
const LOCKED = 423;
const METHOD_FAILURE = 420;
const METHOD_NOT_ALLOWED = 405;
const MOVED_PERMANENTLY = 301;
const MOVED_TEMPORARILY = 302;
const MULTI_STATUS = 207;
const MULTIPLE_CHOICES = 300;
const NETWORK_AUTHENTICATION_REQUIRED = 511;
const NO_CONTENT = 204;
const NON_AUTHORITATIVE_INFORMATION = 203;
const NOT_ACCEPTABLE = 406;
const NOT_FOUND = 404;
const NOT_IMPLEMENTED = 501;
const NOT_MODIFIED = 304;
const OK = 200;
const PARTIAL_CONTENT = 206;
const PAYMENT_REQUIRED = 402;
const PERMANENT_REDIRECT = 308;
const PRECONDITION_FAILED = 412;
const PRECONDITION_REQUIRED = 428;
const PROCESSING = 102;
const EARLY_HINTS = 103;
const UPGRADE_REQUIRED = 426;
const PROXY_AUTHENTICATION_REQUIRED = 407;
const REQUEST_HEADER_FIELDS_TOO_LARGE = 431;
const REQUEST_TIMEOUT = 408;
const REQUEST_TOO_LONG = 413;
const REQUEST_URI_TOO_LONG = 414;
const REQUESTED_RANGE_NOT_SATISFIABLE = 416;
const RESET_CONTENT = 205;
const SEE_OTHER = 303;
const SERVICE_UNAVAILABLE = 503;
const SWITCHING_PROTOCOLS = 101;
const TEMPORARY_REDIRECT = 307;
const TOO_MANY_REQUESTS = 429;
const UNAUTHORIZED = 401;
const UNAVAILABLE_FOR_LEGAL_REASONS = 451;
const UNPROCESSABLE_ENTITY = 422;
const UNSUPPORTED_MEDIA_TYPE = 415;
const USE_PROXY = 305;
const MISDIRECTED_REQUEST = 421;

exports.ACCEPTED = ACCEPTED;
exports.BAD_GATEWAY = BAD_GATEWAY;
exports.BAD_REQUEST = BAD_REQUEST;
exports.CONFLICT = CONFLICT;
exports.CONTINUE = CONTINUE;
exports.CREATED = CREATED;
exports.EARLY_HINTS = EARLY_HINTS;
exports.EXPECTATION_FAILED = EXPECTATION_FAILED;
exports.FAILED_DEPENDENCY = FAILED_DEPENDENCY;
exports.FORBIDDEN = FORBIDDEN;
exports.GATEWAY_TIMEOUT = GATEWAY_TIMEOUT;
exports.GONE = GONE;
exports.HTTP_VERSION_NOT_SUPPORTED = HTTP_VERSION_NOT_SUPPORTED;
exports.IM_A_TEAPOT = IM_A_TEAPOT;
exports.INSUFFICIENT_SPACE_ON_RESOURCE = INSUFFICIENT_SPACE_ON_RESOURCE;
exports.INSUFFICIENT_STORAGE = INSUFFICIENT_STORAGE;
exports.INTERNAL_SERVER_ERROR = INTERNAL_SERVER_ERROR;
exports.LENGTH_REQUIRED = LENGTH_REQUIRED;
exports.LOCKED = LOCKED;
exports.METHOD_FAILURE = METHOD_FAILURE;
exports.METHOD_NOT_ALLOWED = METHOD_NOT_ALLOWED;
exports.MISDIRECTED_REQUEST = MISDIRECTED_REQUEST;
exports.MOVED_PERMANENTLY = MOVED_PERMANENTLY;
exports.MOVED_TEMPORARILY = MOVED_TEMPORARILY;
exports.MULTIPLE_CHOICES = MULTIPLE_CHOICES;
exports.MULTI_STATUS = MULTI_STATUS;
exports.NETWORK_AUTHENTICATION_REQUIRED = NETWORK_AUTHENTICATION_REQUIRED;
exports.NON_AUTHORITATIVE_INFORMATION = NON_AUTHORITATIVE_INFORMATION;
exports.NOT_ACCEPTABLE = NOT_ACCEPTABLE;
exports.NOT_FOUND = NOT_FOUND;
exports.NOT_IMPLEMENTED = NOT_IMPLEMENTED;
exports.NOT_MODIFIED = NOT_MODIFIED;
exports.NO_CONTENT = NO_CONTENT;
exports.OK = OK;
exports.PARTIAL_CONTENT = PARTIAL_CONTENT;
exports.PAYMENT_REQUIRED = PAYMENT_REQUIRED;
exports.PERMANENT_REDIRECT = PERMANENT_REDIRECT;
exports.PRECONDITION_FAILED = PRECONDITION_FAILED;
exports.PRECONDITION_REQUIRED = PRECONDITION_REQUIRED;
exports.PROCESSING = PROCESSING;
exports.PROXY_AUTHENTICATION_REQUIRED = PROXY_AUTHENTICATION_REQUIRED;
exports.REQUESTED_RANGE_NOT_SATISFIABLE = REQUESTED_RANGE_NOT_SATISFIABLE;
exports.REQUEST_HEADER_FIELDS_TOO_LARGE = REQUEST_HEADER_FIELDS_TOO_LARGE;
exports.REQUEST_TIMEOUT = REQUEST_TIMEOUT;
exports.REQUEST_TOO_LONG = REQUEST_TOO_LONG;
exports.REQUEST_URI_TOO_LONG = REQUEST_URI_TOO_LONG;
exports.RESET_CONTENT = RESET_CONTENT;
exports.SEE_OTHER = SEE_OTHER;
exports.SERVICE_UNAVAILABLE = SERVICE_UNAVAILABLE;
exports.SWITCHING_PROTOCOLS = SWITCHING_PROTOCOLS;
exports.TEMPORARY_REDIRECT = TEMPORARY_REDIRECT;
exports.TOO_MANY_REQUESTS = TOO_MANY_REQUESTS;
exports.UNAUTHORIZED = UNAUTHORIZED;
exports.UNAVAILABLE_FOR_LEGAL_REASONS = UNAVAILABLE_FOR_LEGAL_REASONS;
exports.UNPROCESSABLE_ENTITY = UNPROCESSABLE_ENTITY;
exports.UNSUPPORTED_MEDIA_TYPE = UNSUPPORTED_MEDIA_TYPE;
exports.UPGRADE_REQUIRED = UPGRADE_REQUIRED;
exports.USE_PROXY = USE_PROXY;
