// stoker/dist/http-status-phrases.mjs
const ACCEPTED = "Accepted";
const BAD_GATEWAY = "Bad Gateway";
const BAD_REQUEST = "Bad Request";
const CONFLICT = "Conflict";
const CONTINUE = "Continue";
const CREATED = "Created";
const EXPECTATION_FAILED = "Expectation Failed";
const FAILED_DEPENDENCY = "Failed Dependency";
const FORBIDDEN = "Forbidden";
const GATEWAY_TIMEOUT = "Gateway Timeout";
const GONE = "Gone";
const HTTP_VERSION_NOT_SUPPORTED = "HTTP Version Not Supported";
const IM_A_TEAPOT = "I'm a teapot";
const INSUFFICIENT_SPACE_ON_RESOURCE = "Insufficient Space on Resource";
const INSUFFICIENT_STORAGE = "Insufficient Storage";
const INTERNAL_SERVER_ERROR = "Internal Server Error";
const LENGTH_REQUIRED = "Length Required";
const LOCKED = "Locked";
const METHOD_FAILURE = "Method Failure";
const METHOD_NOT_ALLOWED = "Method Not Allowed";
const MOVED_PERMANENTLY = "Moved Permanently";
const MOVED_TEMPORARILY = "Moved Temporarily";
const MULTI_STATUS = "Multi-Status";
const MULTIPLE_CHOICES = "Multiple Choices";
const NETWORK_AUTHENTICATION_REQUIRED = "Network Authentication Required";
const NO_CONTENT = "No Content";
const NON_AUTHORITATIVE_INFORMATION = "Non Authoritative Information";
const NOT_ACCEPTABLE = "Not Acceptable";
const NOT_FOUND = "Not Found";
const NOT_IMPLEMENTED = "Not Implemented";
const NOT_MODIFIED = "Not Modified";
const OK = "OK";
const PARTIAL_CONTENT = "Partial Content";
const PAYMENT_REQUIRED = "Payment Required";
const PERMANENT_REDIRECT = "Permanent Redirect";
const PRECONDITION_FAILED = "Precondition Failed";
const PRECONDITION_REQUIRED = "Precondition Required";
const PROCESSING = "Processing";
const EARLY_HINTS = "Early Hints";
const UPGRADE_REQUIRED = "Upgrade Required";
const PROXY_AUTHENTICATION_REQUIRED = "Proxy Authentication Required";
const REQUEST_HEADER_FIELDS_TOO_LARGE = "Request Header Fields Too Large";
const REQUEST_TIMEOUT = "Request Timeout";
const REQUEST_TOO_LONG = "Request Entity Too Large";
const REQUEST_URI_TOO_LONG = "Request-URI Too Long";
const REQUESTED_RANGE_NOT_SATISFIABLE = "Requested Range Not Satisfiable";
const RESET_CONTENT = "Reset Content";
const SEE_OTHER = "See Other";
const SERVICE_UNAVAILABLE = "Service Unavailable";
const SWITCHING_PROTOCOLS = "Switching Protocols";
const TEMPORARY_REDIRECT = "Temporary Redirect";
const TOO_MANY_REQUESTS = "Too Many Requests";
const UNAUTHORIZED = "Unauthorized";
const UNAVAILABLE_FOR_LEGAL_REASONS = "Unavailable For Legal Reasons";
const UNPROCESSABLE_ENTITY = "Unprocessable Entity";
const UNSUPPORTED_MEDIA_TYPE = "Unsupported Media Type";
const USE_PROXY = "Use Proxy";
const MISDIRECTED_REQUEST = "Misdirected Request";

export { ACCEPTED, BAD_GATEWAY, BAD_REQUEST, CONFLICT, CONTINUE, CREATED, EARLY_HINTS, EXPECTATION_FAILED, FAILED_DEPENDENCY, FORBIDDEN, GATEWAY_TIMEOUT, GONE, HTTP_VERSION_NOT_SUPPORTED, IM_A_TEAPOT, INSUFFICIENT_SPACE_ON_RESOURCE, INSUFFICIENT_STORAGE, INTERNAL_SERVER_ERROR, LENGTH_REQUIRED, LOCKED, METHOD_FAILURE, METHOD_NOT_ALLOWED, MISDIRECTED_REQUEST, MOVED_PERMANENTLY, MOVED_TEMPORARILY, MULTIPLE_CHOICES, MULTI_STATUS, NETWORK_AUTHENTICATION_REQUIRED, NON_AUTHORITATIVE_INFORMATION, NOT_ACCEPTABLE, NOT_FOUND, NOT_IMPLEMENTED, NOT_MODIFIED, NO_CONTENT, OK, PARTIAL_CONTENT, PAYMENT_REQUIRED, PERMANENT_REDIRECT, PRECONDITION_FAILED, PRECONDITION_REQUIRED, PROCESSING, PROXY_AUTHENTICATION_REQUIRED, REQUESTED_RANGE_NOT_SATISFIABLE, REQUEST_HEADER_FIELDS_TOO_LARGE, REQUEST_TIMEOUT, REQUEST_TOO_LONG, REQUEST_URI_TOO_LONG, RESET_CONTENT, SEE_OTHER, SERVICE_UNAVAILABLE, SWITCHING_PROTOCOLS, TEMPORARY_REDIRECT, TOO_MANY_REQUESTS, UNAUTHORIZED, UNAVAILABLE_FOR_LEGAL_REASONS, UNPROCESSABLE_ENTITY, UNSUPPORTED_MEDIA_TYPE, UPGRADE_REQUIRED, USE_PROXY };
