// stoker/dist/http-status-phrases.cjs
'use strict';

const ACCEPTED = "Accepted";
const BAD_GATEWAY = "Bad Gateway";
const BAD_REQUEST = "Bad Request";
const CONFLICT = "Conflict";
const CONTINUE = "Continue";
const CREATED = "Created";
const EXPECTATION_FAILED = "Expectation Failed";
const FAILED_DEPENDENCY = "Failed Dependency";
const FORBIDDEN = "Forbidden";
const GATEWAY_TIMEOUT = "Gateway Timeout";
const GONE = "Gone";
const HTTP_VERSION_NOT_SUPPORTED = "HTTP Version Not Supported";
const IM_A_TEAPOT = "I'm a teapot";
const INSUFFICIENT_SPACE_ON_RESOURCE = "Insufficient Space on Resource";
const INSUFFICIENT_STORAGE = "Insufficient Storage";
const INTERNAL_SERVER_ERROR = "Internal Server Error";
const LENGTH_REQUIRED = "Length Required";
const LOCKED = "Locked";
const METHOD_FAILURE = "Method Failure";
const METHOD_NOT_ALLOWED = "Method Not Allowed";
const MOVED_PERMANENTLY = "Moved Permanently";
const MOVED_TEMPORARILY = "Moved Temporarily";
const MULTI_STATUS = "Multi-Status";
const MULTIPLE_CHOICES = "Multiple Choices";
const NETWORK_AUTHENTICATION_REQUIRED = "Network Authentication Required";
const NO_CONTENT = "No Content";
const NON_AUTHORITATIVE_INFORMATION = "Non Authoritative Information";
const NOT_ACCEPTABLE = "Not Acceptable";
const NOT_FOUND = "Not Found";
const NOT_IMPLEMENTED = "Not Implemented";
const NOT_MODIFIED = "Not Modified";
const OK = "OK";
const PARTIAL_CONTENT = "Partial Content";
const PAYMENT_REQUIRED = "Payment Required";
const PERMANENT_REDIRECT = "Permanent Redirect";
const PRECONDITION_FAILED = "Precondition Failed";
const PRECONDITION_REQUIRED = "Precondition Required";
const PROCESSING = "Processing";
const EARLY_HINTS = "Early Hints";
const UPGRADE_REQUIRED = "Upgrade Required";
const PROXY_AUTHENTICATION_REQUIRED = "Proxy Authentication Required";
const REQUEST_HEADER_FIELDS_TOO_LARGE = "Request Header Fields Too Large";
const REQUEST_TIMEOUT = "Request Timeout";
const REQUEST_TOO_LONG = "Request Entity Too Large";
const REQUEST_URI_TOO_LONG = "Request-URI Too Long";
const REQUESTED_RANGE_NOT_SATISFIABLE = "Requested Range Not Satisfiable";
const RESET_CONTENT = "Reset Content";
const SEE_OTHER = "See Other";
const SERVICE_UNAVAILABLE = "Service Unavailable";
const SWITCHING_PROTOCOLS = "Switching Protocols";
const TEMPORARY_REDIRECT = "Temporary Redirect";
const TOO_MANY_REQUESTS = "Too Many Requests";
const UNAUTHORIZED = "Unauthorized";
const UNAVAILABLE_FOR_LEGAL_REASONS = "Unavailable For Legal Reasons";
const UNPROCESSABLE_ENTITY = "Unprocessable Entity";
const UNSUPPORTED_MEDIA_TYPE = "Unsupported Media Type";
const USE_PROXY = "Use Proxy";
const MISDIRECTED_REQUEST = "Misdirected Request";

exports.ACCEPTED = ACCEPTED;
exports.BAD_GATEWAY = BAD_GATEWAY;
exports.BAD_REQUEST = BAD_REQUEST;
exports.CONFLICT = CONFLICT;
exports.CONTINUE = CONTINUE;
exports.CREATED = CREATED;
exports.EARLY_HINTS = EARLY_HINTS;
exports.EXPECTATION_FAILED = EXPECTATION_FAILED;
exports.FAILED_DEPENDENCY = FAILED_DEPENDENCY;
exports.FORBIDDEN = FORBIDDEN;
exports.GATEWAY_TIMEOUT = GATEWAY_TIMEOUT;
exports.GONE = GONE;
exports.HTTP_VERSION_NOT_SUPPORTED = HTTP_VERSION_NOT_SUPPORTED;
exports.IM_A_TEAPOT = IM_A_TEAPOT;
exports.INSUFFICIENT_SPACE_ON_RESOURCE = INSUFFICIENT_SPACE_ON_RESOURCE;
exports.INSUFFICIENT_STORAGE = INSUFFICIENT_STORAGE;
exports.INTERNAL_SERVER_ERROR = INTERNAL_SERVER_ERROR;
exports.LENGTH_REQUIRED = LENGTH_REQUIRED;
exports.LOCKED = LOCKED;
exports.METHOD_FAILURE = METHOD_FAILURE;
exports.METHOD_NOT_ALLOWED = METHOD_NOT_ALLOWED;
exports.MISDIRECTED_REQUEST = MISDIRECTED_REQUEST;
exports.MOVED_PERMANENTLY = MOVED_PERMANENTLY;
exports.MOVED_TEMPORARILY = MOVED_TEMPORARILY;
exports.MULTIPLE_CHOICES = MULTIPLE_CHOICES;
exports.MULTI_STATUS = MULTI_STATUS;
exports.NETWORK_AUTHENTICATION_REQUIRED = NETWORK_AUTHENTICATION_REQUIRED;
exports.NON_AUTHORITATIVE_INFORMATION = NON_AUTHORITATIVE_INFORMATION;
exports.NOT_ACCEPTABLE = NOT_ACCEPTABLE;
exports.NOT_FOUND = NOT_FOUND;
exports.NOT_IMPLEMENTED = NOT_IMPLEMENTED;
exports.NOT_MODIFIED = NOT_MODIFIED;
exports.NO_CONTENT = NO_CONTENT;
exports.OK = OK;
exports.PARTIAL_CONTENT = PARTIAL_CONTENT;
exports.PAYMENT_REQUIRED = PAYMENT_REQUIRED;
exports.PERMANENT_REDIRECT = PERMANENT_REDIRECT;
exports.PRECONDITION_FAILED = PRECONDITION_FAILED;
exports.PRECONDITION_REQUIRED = PRECONDITION_REQUIRED;
exports.PROCESSING = PROCESSING;
exports.PROXY_AUTHENTICATION_REQUIRED = PROXY_AUTHENTICATION_REQUIRED;
exports.REQUESTED_RANGE_NOT_SATISFIABLE = REQUESTED_RANGE_NOT_SATISFIABLE;
exports.REQUEST_HEADER_FIELDS_TOO_LARGE = REQUEST_HEADER_FIELDS_TOO_LARGE;
exports.REQUEST_TIMEOUT = REQUEST_TIMEOUT;
exports.REQUEST_TOO_LONG = REQUEST_TOO_LONG;
exports.REQUEST_URI_TOO_LONG = REQUEST_URI_TOO_LONG;
exports.RESET_CONTENT = RESET_CONTENT;
exports.SEE_OTHER = SEE_OTHER;
exports.SERVICE_UNAVAILABLE = SERVICE_UNAVAILABLE;
exports.SWITCHING_PROTOCOLS = SWITCHING_PROTOCOLS;
exports.TEMPORARY_REDIRECT = TEMPORARY_REDIRECT;
exports.TOO_MANY_REQUESTS = TOO_MANY_REQUESTS;
exports.UNAUTHORIZED = UNAUTHORIZED;
exports.UNAVAILABLE_FOR_LEGAL_REASONS = UNAVAILABLE_FOR_LEGAL_REASONS;
exports.UNPROCESSABLE_ENTITY = UNPROCESSABLE_ENTITY;
exports.UNSUPPORTED_MEDIA_TYPE = UNSUPPORTED_MEDIA_TYPE;
exports.UPGRADE_REQUIRED = UPGRADE_REQUIRED;
exports.USE_PROXY = USE_PROXY;
