// stoker/dist/http-status-codes.mjs
const ACCEPTED = 202;
const BAD_GATEWAY = 502;
const BAD_REQUEST = 400;
const CONFLICT = 409;
const CONTINUE = 100;
const CREATED = 201;
const EXPECTATION_FAILED = 417;
const FAILED_DEPENDENCY = 424;
const FORBIDDEN = 403;
const GATEWAY_TIMEOUT = 504;
const GONE = 410;
const HTTP_VERSION_NOT_SUPPORTED = 505;
const IM_A_TEAPOT = 418;
const INSUFFICIENT_SPACE_ON_RESOURCE = 419;
const INSUFFICIENT_STORAGE = 507;
const INTERNAL_SERVER_ERROR = 500;
const LENGTH_REQUIRED = 411;
const LOCKED = 423;
const METHOD_FAILURE = 420;
const METHOD_NOT_ALLOWED = 405;
const MOVED_PERMANENTLY = 301;
const MOVED_TEMPORARILY = 302;
const MULTI_STATUS = 207;
const MULTIPLE_CHOICES = 300;
const NETWORK_AUTHENTICATION_REQUIRED = 511;
const NO_CONTENT = 204;
const NON_AUTHORITATIVE_INFORMATION = 203;
const NOT_ACCEPTABLE = 406;
const NOT_FOUND = 404;
const NOT_IMPLEMENTED = 501;
const NOT_MODIFIED = 304;
const OK = 200;
const PARTIAL_CONTENT = 206;
const PAYMENT_REQUIRED = 402;
const PERMANENT_REDIRECT = 308;
const PRECONDITION_FAILED = 412;
const PRECONDITION_REQUIRED = 428;
const PROCESSING = 102;
const EARLY_HINTS = 103;
const UPGRADE_REQUIRED = 426;
const PROXY_AUTHENTICATION_REQUIRED = 407;
const REQUEST_HEADER_FIELDS_TOO_LARGE = 431;
const REQUEST_TIMEOUT = 408;
const REQUEST_TOO_LONG = 413;
const REQUEST_URI_TOO_LONG = 414;
const REQUESTED_RANGE_NOT_SATISFIABLE = 416;
const RESET_CONTENT = 205;
const SEE_OTHER = 303;
const SERVICE_UNAVAILABLE = 503;
const SWITCHING_PROTOCOLS = 101;
const TEMPORARY_REDIRECT = 307;
const TOO_MANY_REQUESTS = 429;
const UNAUTHORIZED = 401;
const UNAVAILABLE_FOR_LEGAL_REASONS = 451;
const UNPROCESSABLE_ENTITY = 422;
const UNSUPPORTED_MEDIA_TYPE = 415;
const USE_PROXY = 305;
const MISDIRECTED_REQUEST = 421;

export { ACCEPTED, BAD_GATEWAY, BAD_REQUEST, CONFLICT, CONTINUE, CREATED, EARLY_HINTS, EXPECTATION_FAILED, FAILED_DEPENDENCY, FORBIDDEN, GATEWAY_TIMEOUT, GONE, HTTP_VERSION_NOT_SUPPORTED, IM_A_TEAPOT, INSUFFICIENT_SPACE_ON_RESOURCE, INSUFFICIENT_STORAGE, INTERNAL_SERVER_ERROR, LENGTH_REQUIRED, LOCKED, METHOD_FAILURE, METHOD_NOT_ALLOWED, MISDIRECTED_REQUEST, MOVED_PERMANENTLY, MOVED_TEMPORARILY, MULTIPLE_CHOICES, MULTI_STATUS, NETWORK_AUTHENTICATION_REQUIRED, NON_AUTHORITATIVE_INFORMATION, NOT_ACCEPTABLE, NOT_FOUND, NOT_IMPLEMENTED, NOT_MODIFIED, NO_CONTENT, OK, PARTIAL_CONTENT, PAYMENT_REQUIRED, PERMANENT_REDIRECT, PRECONDITION_FAILED, PRECONDITION_REQUIRED, PROCESSING, PROXY_AUTHENTICATION_REQUIRED, REQUESTED_RANGE_NOT_SATISFIABLE, REQUEST_HEADER_FIELDS_TOO_LARGE, REQUEST_TIMEOUT, REQUEST_TOO_LONG, REQUEST_URI_TOO_LONG, RESET_CONTENT, SEE_OTHER, SERVICE_UNAVAILABLE, SWITCHING_PROTOCOLS, TEMPORARY_REDIRECT, TOO_MANY_REQUESTS, UNAUTHORIZED, UNAVAILABLE_FOR_LEGAL_REASONS, UNPROCESSABLE_ENTITY, UNSUPPORTED_MEDIA_TYPE, UPGRADE_REQUIRED, USE_PROXY };
